Code/Engine/Foundation/Math/Easing.h

```diff
@@ -0,0 +1,147 @@
+#pragma once
+
+#include <Foundation/Basics.h>
+#include <Foundation/Math/Declarations.h>
+
+namespace ezMath
+{
+  enum ezEasingFunctions
+  {
```

Comment:
Could add a comment where the code is from and if available, add a link to a page that shows the different functions in action.

Reply:
Updated somewhat, I'd love to hear your opinion on it though.

Reply:
Yep, this is useful!